### ISSUE TITLE
Visual Effects Missing in VS2015

### ISSUE THREAD

**Issue by squatched** [LiamMorrow/Visual-Studio-Power-Mode#41]
This has no effect in VS2015. Settings show up but turning it off/on accomplishes nothing.

**Comment by Sekamelica** [comment]
Same for me ! It looks so exciting, I doesn't want to code without this anymore :(.

**Comment by LiamMorrow** [comment]
Yeah not sure what the go is there.  Having a look at a fix

**Comment by sdujs** [comment]
Had this for VS2013 @ VS2015, then both quit working.   Everything was there but would not work, even uninstalled/reinstalled - nothing.   Only changes I recall are various updates from MS, which I did not track.   On a windows 7 64bit machine.   I miss this extension for some odd reason!   Got me to sign up for github...

**Comment by LiamMorrow** [comment]
Yeah it's definitely a problem on the extensions end.  First it didn't work for 2017, then I updated the dependencies and it doesn't work on 2015 :S

**Comment by SelfishOlex** [comment]
The extension broke in VS2015 during one of VS updates. It must have been some ninja change that they didn't tell you about. :P

**Comment by ipepta** [comment]
Before this bug is fixed , you can use the version committed on Feb 9, 2016. 
VS2015 can open it, compile it, install it, and it will work. 
Works for me anyway :)

**Comment by tyh123ooo** [comment]
I cannnot open the Feb 9,2016 version.
I install the product,but doesn't work either.
And how can i uninstall it?

**Comment by januszem** [comment]
@ipepta Is this works only for the first time after instalation for you too?

**Comment by ipepta** [comment]
@januszem  
Disable all extension auto update in Tools->Option. Remove power mode and install it again.
![1](https://user-images.githubusercontent.com/7076584/28030802-d41a7bba-65d7-11e7-8b80-2ff877edadcf.png)

@tyh123ooo 
And here is how to remove power mode. It's in Tools->Extension and Update. Click the uninstall button at the right corner.
![2](https://user-images.githubusercontent.com/7076584/28030973-594ebbb6-65d8-11e7-96c3-ff8fb0831cde.png)

**Comment by ipepta** [comment]
This is the version I'm using. 
It's compiled from the code committed on Feb 9, 2016. Using VS 2015 chinese language version. 
Disable all extension auto update before install.

[PowerMode_Feb_9_2016.zip](https://github.com/LiamMorrow/Visual-Studio-Power-Mode/files/1136503/PowerMode_Feb_9_2016.zip)

**Comment by sdujs** [comment]
Worked on vs2013 but not vs2015.   Thanks for the zip ...

**Comment by ipepta** [comment]
@sdujs 
That's strange. I'm using it myself and it's working very well....
![3](https://user-images.githubusercontent.com/7076584/28033225-0525f768-65e0-11e7-97fb-3cc33f50c4ad.png)

**Comment by tyh123ooo** [comment]
@ipepta  thank u.I uninstall it.And I will try ur version,see if it can work or not

**Comment by tyh123ooo** [comment]
@ipepta  thx，this version works. Why my git desktop cannot revert codes in win7? I remember i can do it in win10

**Comment by ipepta** [comment]
@tyh123ooo  Don't know how to use git desktop....personally I prefer SVN. I find the old version code from git website XD.

**Comment by januszem** [comment]
@ipepta Thanks! Autoupdates was the thing.
